### ISSUE TITLE
stdc++fs isn't needed on FreeBSD

### ISSUE THREAD

**Issue by yurivict** [octobanana/peaclock#12]
It isn't found.

**Comment by octobanana** [comment]
Could you give some more details on the issue?

What is the compiler/version being used?  
What is the error output from the build?  
If you remove the `stdc++fs` line from __CMake__, does it then work?

The __GCC__ implementation of `std::filesystem` requires linking with `stdc++fs` and the __LLVM__ implementation requires linking with `c++fs`.

**Comment by yurivict** [comment]
FreeBSD 12 amd64, clang-8.
There is no ```stdc++fs``` present in the system for clang.
Only gcc installs ```/usr/local/lib/gcc8/libstdc++fs.a```, and then build fails with missing library.

**Comment by octobanana** [comment]
Using __clang-8__, could you try making the following change in the `CMakeLists.txt` file and re-build:

```diff
 target_link_libraries (
   ${TARGET}
   pthread
-  stdc++fs
+  c++fs
   icuuc
   icui18n
 )
```

**Comment by yurivict** [comment]
I just removed ```stdc++fs``` and it builds.

**Comment by octobanana** [comment]
Fixed in version [0.4.1](https://github.com/octobanana/peaclock/releases/tag/0.4.1), thanks!

**Comment by rajhlinux** [comment]
I also had the same issue on FreeBSD 13.1

Note: "c++fs" does not work either.

I had to completely remove "stdc++fs" from CMakeList.txt and rebuild. 
Worked for me. Using it for a different project git.

Thanks.
